pages/[category]/index.tsx

```diff
@@ -1,51 +1,236 @@
-import { useEffect } from 'react'
+import { useEffect, useState } from 'react'
 import Head from 'next/head'
 import Error from 'next/error'
 import { useRouter } from 'next/router'
-
+import { Fab, Modal, Hidden, Paper, Grid } from '@material-ui/core/'
+import { FilterList, List, Room } from '@material-ui/icons/'
+import withWidth, { WithWidth } from '@material-ui/core/withWidth'
 import {
   useMultipleListRecords,
   useConvertedLocationRecords,
   useLanguage,
 } from '../../hooks/'
-import { siteTitle, categories } from '../../constants'
-import { RecordPane, DisplayMap } from '../../components'
-
-const LandingPage = () => {
+import {
+  RecordPane,
+  DisplayMap,
+  CategoryFilters,
+  MobileFilterModal,
+} from '../../components'
+import {
+  categoryCopy,
+  getMatchingRecords,
+  siteTitle,
+  categories,
+  useStyles,
+} from '../../constants/'
+import { Title, Paragraph } from '../../ui'
+const ITEM_HEIGHT = 48
```

Comment:
I think that relying on absolute units is not going to serve us well for devices of varying sizes. Is there a way you can do this is `flexbox` or `grid`?